src/settings/FinePolicy/components/EditSections/FinesSection/FinesSection.js

```diff
@@ -42,12 +41,12 @@ class FinesSection extends React.Component {
         <Accordion
           id="editFineSection"
           open={fineSectionOpen}
-          label={<FormattedMessage id="ui-circulation.settings.finePolicy.overdueFine" />}
+          label={intl.formatMessage({ id: 'ui-circulation.settings.finePolicy.overdueFine' })}
```

Comment:
@Dmitriy-Litvinenko, is it absolutely safe for us?

Reply:
I've checked all components where I changed passed label prop and it is safe. And I changed props types in that components as well